examples/chip-tool/templates/partials/test_cluster.zapt

```diff
@@ -142,18 +142,8 @@ class {{filename}}: public TestCommand
         {{#if async}}ReturnErrorOnFailure({{else}}return {{/if}}cluster.InvokeCommand<requestType, responseType>(request, this, success, failure){{#if async}}){{/if}};
         {{else}}
         {{#chip_tests_item_parameters}}
-        {{chipType}} {{asLowerCamelCase name}}Argument =
-          {{#if (isOctetString type)}}
-            chip::ByteSpan(chip::Uint8::from_const_char("{{definedValue}}"), strlen("{{definedValue}}"))
-          {{else if (isCharString type)}}
-            chip::CharSpan("{{definedValue}}", strlen("{{definedValue}}"))
-          {{else}}
-            {{#if_chip_enum type}}
-              static_cast<{{chipType}}>({{definedValue}}{{asTypeLiteralSuffix type}})
-            {{else}}
-              {{definedValue}}{{asTypeLiteralSuffix type}}
-            {{/if_chip_enum}}
-          {{/if}};
+        {{zapTypeToEncodableClusterObjectType type ns=parent.cluster}} {{asLowerCamelCase name}}Argument;
+        {{>commandValue ns=parent.cluster ignore=true container=(concat (asLowerCamelCase name) "Argument") definedValue=definedValue}}
```

Comment:
This mandate all commandValue implement '=' operator, such as:

        chip::app::DataModel::List<const uint8_t> listInt8uArgument;
        listInt8uArgument = {1, 2, 3, 4};

Reply:
Yes, and they do.